.github/workflows/ci.yml

```diff
@@ -118,7 +136,7 @@ jobs:
       - name: Install Python3
         uses: actions/setup-python@v2
         with:
-          python-version: '3.7.10'
+          python-version: '3.8'
```

Comment:
This python block should be left alone or removed. It was added for a security patch, which is now available in other github python versions so can be removed or left for when we clean up all the python version locks from that.

Reply:
Adding a version tag of "3.8" will automatically update the version of python used to the latest 3.8.x release.

Reply:
Yes, removing it will use whatever the version shipped with ubuntu is which is fine for memory estimator. No need to lock it to 3.8

Reply:
I prefer to specify a specific python version so that we don't run into compatibility breaks later on.

Reply:
The goal of this PR is not updating Python version also we need to be consistent with whatever approach we take.  If we have to change it, it needs to be done in all other repositories as well. Let's address it in a separate PR.

Reply:
Alright. I'll leave that as an exercise to anyone who wants to deal with it.